app/api/question_routes.py

```diff
@@ -0,0 +1,25 @@
+from flask import Blueprint
+from flask_login import current_user, login_required
+from app.models import Question, User
+
+question_routes = Blueprint('questions', __name__)
+
+@question_routes.route('/')
+def questions():
+    """
+    Query for all questions and returns them in a list of question dictionaries
+    """
+    questions = Question.query.all()
+    return {'questions': [question.to_dict() for question in questions]}
+
+@question_routes.route('/current')
+# @login_required
+def user_questions():
+    """
+    Query for all of the current users questions and returns them in
+    a list of question dictionaries
+    """
+    user_id = current_user.get_id()
+    # user_id = 1 #! Hard coded in for testing without login, remove once unneeded
+    questions = Question.query.filter(Question.user.has(id=user_id))
```

Comment:
Is this comparing Question.id to user.id?  I think we need to compare Question.user_id to user_id

Reply:
This does the same check because it checks the attached user through the relationship. But Question.user_id would also work the same. It has been tested and works